web/src/settings_org.js

```diff
@@ -702,29 +706,191 @@ export function discard_property_element_changes(elem, for_realm_default_setting
     update_dependent_subsettings(property_name);
 }
 
+export function valid_to(deletes_in) {
+    const time_valid = Number.parseFloat(deletes_in);
+    if (time_valid === 0) {
+        return $t({defaultMessage: "Data will be deleted immediately"});
+    }
+
+    if (!time_valid) {
+        return $t({defaultMessage: "Never deleted"});
```

Comment:
Can we modify this text to be "Data would never be deleted"?

Reply:
`Data will not be automatically deleted` is perhaps more accurate?

Reply:
I have modified the text to use `Data will not be automatically deleted`.